### PR TITLE
Feature/branding layout fixes

### PR DESCRIPTION
**Pull request by Malet**

### CODE REVIEW

File: lib/conduit_web/templates/layout/app.html.eex [hunk 1]
```diff
@@ -2,33 +2,55 @@
...
+      </button>
+      <a class="navbar-brand" href="">
+        <img height="19px" src="<%= static_path(@conn, "/images/conduit-white.svg") %>" alt="Conduit CI"/>
+      </a>
+
+      <div class="collapse navbar-collapse" id="navbarsExampleDefault">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item<%#= active_class(@conn, pipeline_path(@conn, :index))%>">
+            <a class="nav-link" href="<%#= pipeline_path(@conn, :index) %>">Pipelines</a>
+          </li>
```
Comment (IanLuites) [inline lib/conduit_web/templates/layout/app.html.eex:29]: Maybe it would be useful to create a helper that combines the active and the `a` generation?

**Review by IanLuites — APPROVED** [review]
Looks good! 👍🏻 2 small comments :D

File: lib/conduit_web/templates/layout/app.html.eex (hunk 1 above)
Comment (Malet) [reply lib/conduit_web/templates/layout/app.html.eex:29]: personally I think thats overkill though, as we still want to see the markup in the template more than the views

Comment (IanLuites) [reply lib/conduit_web/templates/layout/app.html.eex:29]: Fair enough, is too early for things like that anyway.

**Comment by Malet** [comment]
Are we expecting any amount of large images or binary data though?

**Comment by IanLuites** [comment]
Actually, don't think so. Most branding will probably be that svg and some icons. Most shall probably CSS.

**Pull request merged**